tests/bluechi_test/bluechi_is_online.py

```diff
@@ -52,3 +51,12 @@ def system_is_online(self) -> bool:
             0,
         )
         return result == 0
+
+    def agent_is_online_with_wait(self, wait_time: int) -> bool:
```

Comment:
Not necessary to have a new method for `wait` parameter, we can manage to have a single method, something like below will allow to have a single method with various input parameters:

```
 def agent_is_online(self, wait: int = None) -> bool:
        cmd = [ "agent"]

        if wait:
            cmd.append("--wait")
            cmd.append(f"{wait}")

        result, output = self.run(
            "Checking if agent is active.",
            " ".join(cmd),
            False,
            0,
        )
        return result == 0
```

Reply:
You are right! Thank you :)